stock_orderpoint_manual_procurement/wizards/make_procurement_orderpoint.py

```diff
@@ -71,9 +71,7 @@ def make_procurement(self):
             if not item.orderpoint_id:
                 raise ValidationError(_("No reordering rule found!"))
             values = item.orderpoint_id._prepare_procurement_values()
-            values["date_planned"] = fields.Datetime.to_string(
-                fields.Date.from_string(item.date_planned)
-            )
+            values["date_planned"] = fields.Date.from_string(item.date_planned)
```

Comment:
This is probably not needed. `item.date_planned` should already be a date, no?

```suggestion
            values["date_planned"] = item.date_planned
```

Reply:
Good catch, applied suggested :+1: